erp/management/commands/export_to_datagouv.py

```diff
@@ -1,12 +1,50 @@
+import requests
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
-from erp.jobs import export_to_datagouv
+from erp.export.export import export_schema_to_csv, upload_to_datagouv
+from erp.export.mappers import EtalabMapper
+from erp.models import Erp
 
 
 class Command(BaseCommand):
+    help = "Exporte vers datagouv"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Afficher les erreurs",
+        )
+
     def handle(self, *args, **options):
-        try:
-            export_to_datagouv.job()
-            print("Data exported successfully to 'export.csv'")
-        except KeyboardInterrupt:
-            print("Interrompu.")
+        verbose = options.get("verbose", False)
+        csv_path = "acceslibre.csv"
+        with open(csv_path, "w", newline="") as csv_file:
+            try:
+                erps = Erp.objects.published().having_a11y_data()
+                export_schema_to_csv(csv_file, erps, EtalabMapper)
+                if verbose:
+                    print("Local export successful: 'acceslibre.csv'")
+                if settings.DATAGOUV_API_KEY:
```

Comment:
Here's a general idea: how about we only publish to datagouv if the `DATAGOUV_API_KEY` is defined? that way we don't need to declare environments, just setting an env var or not.